spring-aop/src/main/java/org/springframework/aop/interceptor/AsyncExecutionAspectSupport.java

```diff
@@ -167,6 +174,9 @@ protected AsyncTaskExecutor determineAsyncExecutor(Method method) {
 		if (executor == null) {
 			Executor targetExecutor;
 			String qualifier = getExecutorQualifier(method);
+			if (stringValueResolver != null && StringUtils.hasLength(qualifier)) {
+				qualifier = stringValueResolver.resolveStringValue(qualifier);
```

Comment:
This change currently [fails the build](https://ci.spring.io/builds/190047#L62728b6d:254:256).

```suggestion
			if (this.stringValueResolver != null && StringUtils.hasLength(qualifier)) {
				qualifier = this.stringValueResolver.resolveStringValue(qualifier);
```